app/src/main/res/anim/hint_bulb_animation.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<set xmlns:android="http://schemas.android.com/apk/res/android"
+  android:fillAfter="true"
+  android:interpolator="@android:anim/bounce_interpolator">
+  <alpha
+    android:duration="600"
+    android:fromAlpha="0.0"
+    android:interpolator="@android:anim/accelerate_interpolator"
```

Comment:
Why an accelerator interpolator? To have a bit of a "pulse"?

Reply:
I've changed the interpolation logic from the original solution, so this comment is no longer relevant.